mvnw

```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```

Comment:
TODO skimmed this, seems legit, I am however not in the mood to read closely, read closely later

Reply:
seems legit to me...